Assets/__Scenes/04_Options.unity

```diff
@@ -16541,6 +16669,440 @@ RectTransform:
     type: 3}
   m_PrefabInstance: {fileID: 820955159}
   m_PrefabAsset: {fileID: 0}
+--- !u!1001 &835319594
```

Comment:
I know it's not obvious but this object needs adding to the SearchableSection in its parent. Probably should add an option to that class that allows it to populate at runtime but that's outside the scope of this.